dockstore-common/src/main/java/io/dockstore/common/EntryType.java

```diff
@@ -8,5 +8,6 @@
 public enum EntryType {
     TOOL,
     WORKFLOW,
-    SERVICE
+    SERVICE,
+    GITHUBAPPTOOL
```

Comment:
Kinda liked it without specifying github on the off-chance we create these with, say, a gitlab app someday